app/domain/service/burn-event.go

```diff
@@ -23,4 +23,6 @@ type BurnEvent interface {
 	// ProcessEvent processes the burn event by submitting the appropriate
 	// scheduled transaction, leaving the synchronization of the actual transfer on HCS
 	ProcessEvent(event burn_event.BurnEvent)
+	// TransactionID returns the corresponding transaction id
```

Comment:
nit: This inline doc seems confusing. Maybe this would be better:
`TransactionID returns the corresponding Scheduled Transaction paying out the fees to validators and the amount being bridged to the receiver address`